docs/add_data_samples.md

```diff
@@ -0,0 +1,12 @@
+# Add data samples to Substra
+
+Data samples can be ingested to Substra using our python SDK or our command line
+interface tool.
+
+2 different modes are available when adding data samples:
+- local: data are located on the local filesystem and are transferred through HTTP
+  requests. This mode is most likely to be used to add relatively small files or for
+  testing purposes.
+- remote: data are located on the server filesystem. Data must have been transferred
+  previously to the server. This mode is recommended for adding relatively big data 
```

Comment:
`Data must have been transferred previously to the server.` could not be clear for the user. You can maybe develop this part ?

Reply:
Yes, good idea. Do you have some suggestions to develop this part?

Reply:
Something like you may need to ask the administrator of the substra node how to transfer your data. It could be ssh ou sftp or another protocol

Reply:
Thanks. Done.